### ISSUE TITLE
Consistent labels for <nav> elements

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1127]
A screen reader user can navigate a page by region. One of these regions is navigation `<nav>`. When more than one nav element exists on a page, `aria-label` or `aria-labelledby` should be used to identify the type of navigation to the user. 

For example, the following element would be announced in JAWS as "Breadcrumb navigation region":
`<nav aria-label="Breadcrumb">`

Additionally, the component examples we create should conform to a consistent set of labels, since many of these examples will be copied and pasted with a high potential of text strings never being updated afterward. Therefore, strings we define in examples should be ready to use out of the box.

Updates for this issue:

- Of the components that can be wrapped with `<nav>`, the following are strings that should be defined in an `aria-label` attribute:
  - main Nav component: "Global"
  - tertiary Nav: "Local" 
  - tabs: "Local"
    - Note that tabs are not required to use a wrapping `<nav>` element. The documentation for Tabs should note that using `<nav>` with this aria-label should only be used for a tab list that is unique on the current page. Cases where the same set of tabs are duplicated in multiple regions on a page (e.g. cards on a dashboard), are less likely to benefit from using  `<nav>` element.
  - breadcrumbs: "Breadcrumb"
  - pagination: "Pagination"
- Additionally, in the Page component, the hamburger button includes a label that references the primary nav region. These labels should be consistent. 
  - For the element `.pf-c-page__header-brand-toggle > .pf-c-button`, change the `aria-label` value to "Global navigation"

**Comment by jgiardino** [comment]
@mcarrano @stacymcauliffe Could you review this issue and the strings and let me know if you have any suggestions? I wasn't sure if "Application level" would be better than "Product level", or if that string should include a hyphen? Any other suggestions are welcome, too. :-)

**Comment by mcarrano** [comment]
@jgiardino not sure what the right answer is here, but I would definitely avoid "Product." In UX speak, the terms that I would use would be "Global" for the applications level nav and "Local" for the page level nav. But I'm not sure these would be meaningful to the user. @stacymcauliffe what do you think?

**Comment by stacymcauliffe** [comment]
@mcarrano @jgiardino -- 

In the design guidelines, we refer to the different types of nav: 
Primary nav is that application-level global nav
Secondary nav is the second level of the global nav  
Tertiary nav is the third level of navigation specific to a page

So, we use primary, secondary, tertiary - but then we explain what that means.

For aria labels, we could say: Primary, Secondary, and Tertiary 
OR 
We could say : Global, Secondary global, and Local
OR 
**We could just say: Global (to refer to all levels of the global nav) and Local (for the page-specific nav), which is probably the best option.** 

If we go with that option, I will update the design guidelines so the terminology is consistent. 

Thanks!

**Comment by jgiardino** [comment]
Thanks, @stacymcauliffe!

> We could just say: Global (to refer to all levels of the global nav) and Local (for the page-specific nav), which is probably the best option.

I think this is the option that works best with how these components are implemented.

To make sure we're talking about the same things, when I think of primary and secondary nav, I'm thinking of the following example, where "Link 1" is a primary nav item, and "Current Link" is a secondary nav item.

If that is not what you were thinking, let me know. But if so, then in this case, both sets of links would be wrapped in the same `<nav>` element and therefore would have the same label. 

I updated the original description with these links, but can change again if needed. :-)

![image](https://user-images.githubusercontent.com/21063328/50772082-8f2da280-125a-11e9-96a5-a394ed73c5e3.png)

**Comment by stacymcauliffe** [comment]
Yep @jgiardino - we're on the same page about the primary and secondary nav

**Comment by jgiardino** [comment]
Thanks, @stacymcauliffe!

**Comment by jgiardino** [comment]
I was chatting with @stacymcauliffe about aria-label text strings in general, and based on her feedback I just now updated the aria-label value mentioned in the issue description for the hamburger.

**Comment by rebeccaalpert** [comment]
Hey @jgiardino, I'm working on the [corresponding patternfly-react issue](https://github.com/patternfly/patternfly-react/issues/1182) for this. Would tabs be the [Horizontal List and Tertiary List](http://patternfly-react.surge.sh/patternfly-4/components/nav#tertiary-link2) or something else?

**Comment by jgiardino** [comment]
I think this is how these labels would be applied. @mcarrano, please confirm this is correct.

Tabs are being contributed in https://github.com/patternfly/patternfly-react/pull/1144

Nav > Horizontal List would use "Global"
![image](https://user-images.githubusercontent.com/21063328/51599251-716d6800-1eff-11e9-83fd-4d8235cf48ac.png)

Nav > Tertiary  would use "Local"
![image](https://user-images.githubusercontent.com/21063328/51601347-62d57f80-1f04-11e9-9a56-f893f5365f42.png)

**Comment by rebeccaalpert** [comment]
Thanks!

**Comment by mcarrano** [comment]
Sounds right to me @jgiardino @rebeccaalpert
